frontend/src/scenes/data-management/database/databaseTableListLogic.ts

```diff
@@ -12,15 +14,28 @@ export const databaseTableListLogic = kea<databaseTableListLogicType>([
     actions({
         setSearchTerm: (searchTerm: string) => ({ searchTerm }),
     }),
-    loaders({
+    loaders(({ values }) => ({
         database: [
             null as Required<DatabaseSchemaQuery['response']> | null,
             {
                 loadDatabase: async (): Promise<Required<DatabaseSchemaQuery['response']> | null> =>
                     await query({ kind: NodeKind.DatabaseSchemaQuery } as DatabaseSchemaQuery),
             },
         ],
-    }),
+        dataWarehouse: [
```

Comment:
Maybe makes sense to just have a dedicated logic for datawarehouse tables now? Separate from scene based logic

Reply:
Yeah, I also wanna clean up all the logic - as well as clean up the backend api for getting the tables. We're currently calling multiple endpoints to get all the table data we need when really it can be done in one. So, keen to follow this up in a future PR and do some general clean up